### PR TITLE
Pareto Intro Notebook

### PR DESCRIPTION
**Pull request by juanitorduz**
Add a notebook for the Pareto model similar to https://www.pymc-marketing.io/en/stable/notebooks/clv/bg_nbd.html

----
📚 Documentation preview 📚: https://pymc-marketing--627.org.readthedocs.build/en/627/

### CODE REVIEW

File: docs/source/notebooks/clv/pareto_nbd.ipynb [hunk 1]
```diff
@@ -0,0 +1,4408 @@
+{
```
Comment (wd60622) [inline docs/source/notebooks/clv/pareto_nbd.ipynb:1]: <p>*lifetime8 -&gt; <em>lifetime under dropout process 3.?</em> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/627/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

**Comment by ColtAllen** [comment]
We've been doing duplicate work 😬
https://github.com/ColtAllen/pymc-marketing/blob/pnbd_notebook/docs/source/notebooks/clv/pareto_nbd.ipynb

Everything above the **ParetoNBD Model in `pymc_marketing`** header is what I've got so far; the remainder is a bunch of scratchpad code that'll likely get wiped.

**Comment by juanitorduz** [comment]
@ColtAllen I was not sure about this and that is why I pushed this nb. I simply replaced bg -> pareto 
So I can close this one 😅

**Comment by ColtAllen** [comment]
I'll open a PR for mine this week; sorry for the lack of communication on my part.

**Pull request closed without merging**